posthog/special_migrations/setup.py

```diff
@@ -22,6 +23,14 @@
 SPECIAL_MIGRATIONS_MODULE_PATH = "posthog.special_migrations.migrations"
 SPECIAL_MIGRATIONS_EXAMPLE_MODULE_PATH = "posthog.special_migrations.examples"
 
+all_migrations = import_submodules(SPECIAL_MIGRATIONS_MODULE_PATH)
+
+if DEBUG and not TEST:
+    all_migrations["example"] = import_submodules(SPECIAL_MIGRATIONS_EXAMPLE_MODULE_PATH)["example"]
```

Comment:
why are we including the example if running in debug mode? I can run PostHog in debug mode and I'd assume just more logging, but no other changes

Reply:
Unfortunately `DEBUG` has been taken to mean "dev" for a long time now. We use that flag to distinguish a dev environment